src/test/ui/issues/issue-34334.stderr

```diff
@@ -1,10 +1,47 @@
+error: expected expression, found reserved identifier `_`
```

Comment:
This is an ICE regression test, so I'm disinclined to change it to avoid the new errors.